src/cli/bootstrap/bootstrap.ts

```diff
@@ -18,16 +19,27 @@ export default async function bootstrap(args: string[]) {
     const newPublicKey = newPrivateKey.toPublic();
 
     try {
-        await createAntelopeAccount({ account: 'eosio.token' }, signer);
+        await createAntelopeAccount({ account: 'demo.tmy' }, signer);
         await deployContract(
             {
-                account: 'eosio.token',
-                contractDir: path.join(__dirname, '../../Tonomy-Contracts/contracts/eosio.token'),
+                account: 'demo.tmy',
+                contractDir: path.join(__dirname, '../../Tonomy-Contracts/contracts/demo.tmy'),
             },
             signer
         );
-        await eosioTokenContract.create('1000000000 SYS', signer);
-        await eosioTokenContract.issue('10000 SYS', signer);
+        await demoTokenContract.create('1000000000 SYS', signer);
+        await demoTokenContract.issue('10000 SYS', signer);
+
+        await createAntelopeAccount({ account: 'onocoin.tmy' }, signer);
+        await deployContract(
+            {
+                account: 'onocoin.tmy',
+                contractDir: path.join(__dirname, '../../Tonomy-Contracts/contracts/onocoin.tmy'),
+            },
+            signer
+        );
+        await onoCoinContract.create('50000000000.0000 ONO', signer);
+        await onoCoinContract.issue('32500000000.0000 ONO', signer);
 
         await createAntelopeAccount({ account: 'id.tmy' }, signer);
```

Comment:
after all the accounts are created, allocate the coins to them as per slide 23 in the deck https://www.canva.com/design/DAFzI6RvDKA/9RM1pllCtFvFTnbsbSY6XA/edit

account | allocation (% of total supply)
team.tm 15%
ecosystm.tmy 30%
private1.tmy 2.5%
private2.tmy 2.5%
private3.tmy 2.5%
public1.tmy 2.5%
public2.tmy 2.5%
public3.tmy 2.5%
operation.tmy (rename this) 40%

Reply:
if i change the opration.tmy to operation.tmy it throws maximum character exceed error.

Reply:
ah ok. then lets call it `ops.tmy`